### PR TITLE
MP resprite and overhaul 

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Changes up the MP aesthetic to be more akin to Vietnam era MPs, via sprite changes. Command roles no longer require Security hours (up for discussion). SMs are PFCs when they have less than 10 hours in the role, but become Cpls afterwards.

MP Cadets were removed in favour of folding them into MPs having ranks based on playtime, giving MP mains a way to see if a new MP is playing and show them the ropes.

The Warden has been removed as individual MPs can handle these duties on a case-by-case basis, or the CMP can assist.

Lastly, MPs are now provided either a M41A Pulse Rifle MK2 or a L42A Battle Rifle as their standard-issue rifle, as well as the choice of taking a magazine pouch. This will aid in ship and FOB defense.

An MP is just another boot, not a robot, a fellow marine making sure pandemonium doesn't break out on the ship via enforcing marine law. Their duty is to defend and keep order on the ship and the landing zones.

MP:
image

Chief MP:
image

## Why It's Good For The Game

There are various things we'd like to change about MPs to strengthen the bond between them and normal marines. Changing up the aesthetics is a good first step, with effective ML changes to follow, we can make the department a great part of the corps, which contributes both to keeping chaos on the ship to a minimum, and the security of the operation.

## Changelog

:cl: Geeves, Frans_Feiffer, Triiodine
add: Overhauled MP aesthetics. MPs now have standard-issue rifles for ship and FOB defense.
del: Removed Wardens and MP cadets.
del: Command roles no longer need security/MP hours.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
id recommend going ahead and making the brig closer to other parts of the ship to be less isolated and away from marines, and maybe even reducing the red tint the floor and doors and whatever have so it's less isolated too

**Comment by Superreallycoolguy** [comment]
Shipside marine is a stupid name, im not gonna lie G. Keep that shit as Military Police, it doesn't fit with the other ranks. also "Master-At-Arms" is (IM PRETTY SURE) the job title of Enlisted NAVY military police. If we had a whole rank rework to fit within similar IRL USMC ranks, this don't fit.

Everything else is baller tho

File: code/game/machinery/vending/vendor_types/crew/ship_marine.dm [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		list("M4A3 Pistol", 0, /obj/item/storage/belt/gun/m4a3/full, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+		list("Mod 88 Pistol", 0, /obj/item/storage/belt/gun/m4a3/mod88, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+
+		list("ARMOR (CHOOSE 1)", 0, null, null, null),
+		list("Ship Marine M2 Pattern Armor", 0, /obj/item/clothing/suit/storage/marine/ship_marine, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
+		list("Ship Marine M2 Pattern Padless Armor", 0, /obj/item/clothing/suit/storage/marine/ship_marine/padless, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
+
+		list("COMBAT EQUIPMENT (TAKE ALL)", 0, null, null, null),
+		list("Ship Marine M10 Helmet", 0, /obj/item/clothing/head/helmet/marine/ship_marine, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_MANDATORY),
+		list("Marine Combat Boots", 0, /obj/item/clothing/shoes/marine/knife, MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY),
```
Comment (carlarctg) [inline code/game/machinery/vending/vendor_types/crew/ship_marine.dm:21]: boots aren't combat equipment 🤔

**Comment by carlarctg** [comment]
> Shipside marine is a stupid name, im not gonna lie G. Keep that shit as Military Police, it doesn't fit with the other ranks. also "Master-At-Arms" is (IM PRETTY SURE) the job title of Enlisted NAVY military police. If we had a whole rank rework to fit within similar IRL USMC ranks, this don't fit.
> 
> Everything else is baller tho

i significantly prefer shipside marine to military police simply due to the blurring of the division between MPs and marines. if there was a less lame name, though, i would be in favor

File: code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@ var/list/department_radio_keys = list(
 	  ":d" = SQUAD_MARINE_4,			"#d" = SQUAD_MARINE_4,				".d" = SQUAD_MARINE_4,
 	  ":e" = SQUAD_MARINE_5,			"#e" = SQUAD_MARINE_5,				".e" = SQUAD_MARINE_5,
 	  ":f" = SQUAD_MARINE_CRYO,			"#f" = SQUAD_MARINE_CRYO,			".f" = SQUAD_MARINE_CRYO,
-	  ":p" = RADIO_CHANNEL_MP ,			"#p" = RADIO_CHANNEL_MP ,			".p" = RADIO_CHANNEL_MP ,
+	  ":p" = RADIO_CHANNEL_SECURITY ,			"#p" = RADIO_CHANNEL_SECURITY ,			".p" = RADIO_CHANNEL_SECURITY ,
```
Comment (stanalbatross) [inline code/modules/mob/living/say.dm:17]: if it's being called security channel change to .s imo

**Review by stanalbatross — CHANGES_REQUESTED** [review]
and fix your conflicts PLEASE 💢

**Review by nauticall — CHANGES_REQUESTED** [review]
Keep titles to Military Police and CMP to prevent mass-confusion. As people said earlier this can be really easily confused with Squad Rifleman and people won't know the role of these people from a glance.

Also, "Master-at-Arms" is the name for Navy MPs, who are also enlisted. Calling the CMP an MAA would create even more confusion.

**Review by nauticall — CHANGES_REQUESTED** [review]
image
Helmet's white stripe breaks the silhouette. You can add an outline (bottom row) while still preserving the look of the helmet to be distinct.

**Comment by morrowwolf** [comment]
Requesting internal discussion regarding these changes given there has been almost zero communication in the dev side of the house about the vast majority of this until about 6 hours ago.

**Comment by MusselMain** [comment]
Nice, but can they have slightly more visible distinction (in their armour) from standard marines? (it makes it easier for off-ship players to easily identify MP's/Ship Marines, and follow their orders instead of having to examine them to examine them to make sure it's not some marine wearing a MP helmet).

**Comment by MusselMain** [comment]
Hey, if I can't post here, where to I leave make comments on the discord?

**Comment by LoJoFan** [comment]
> Shipside marine is a stupid name, im not gonna lie G. Keep that shit as Military Police, it doesn't fit with the other ranks. also "Master-At-Arms" is (IM PRETTY SURE) the job title of Enlisted NAVY military police. If we had a whole rank rework to fit within similar IRL USMC ranks, this don't fit.
> 
> Everything else is baller tho

I'm pretty sure USCMC ranks were fully refitted to be navy ones months ago.

File: code/game/machinery/vending/vendor_types/crew/mp.dm [hunk 3]
```diff
@@ -1,97 +1,65 @@
...
+		/obj/item/ammo_magazine/rifle/extended,
+		/obj/item/ammo_magazine/rifle,
+		/obj/item/ammo_magazine/rifle
+	)
+
+/obj/effect/essentials_set/mp/l42a
+	spawned_gear_list = list(
+		/obj/item/weapon/gun/rifle/l42a,
+		/obj/item/device/binoculars,
+		/obj/item/ammo_magazine/rifle/l42a/extended,
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/mp.dm:62]: nooooooooooooooooooooooooooooooooo

File: code/modules/gear_presets/uscm_police.dm [hunk 4]
```diff
@@ -27,22 +27,27 @@
...
 	utility_hat = list(/obj/item/clothing/head/helmet/beret/marine/mp)
 	utility_extra = list(/obj/item/clothing/head/cmcap, /obj/item/clothing/head/beret/cm, /obj/item/clothing/head/beret/cm/tan)
 
-	service_over = list(/obj/item/clothing/suit/storage/jacket/marine/service/mp)
+	service_over = list(/obj/item/clothing/suit/storage/jacket/marine/service)
+
+/datum/equipment_preset/uscm_ship/mp/standard/load_rank(mob/living/carbon/human/H)
+	if(H.client && get_job_playtime(H.client, rank) < JOB_PLAYTIME_TIER_1)
+		return "ME2"
+	return paygrade
```
Comment (stanalbatross) [inline code/modules/gear_presets/uscm_police.dm:43]: non-detailed var name detected 💢

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Comment by Sinbest** [comment]
Love the aesthetic changes, the one thing I object to is the helmet, it staying black would be fine.

**Comment by Sinbest** [comment]
This PR would be far better without the pointless changes to MP roles and such.

**Comment by SeeminglyTypicalUsername** [comment]
The armor they get in-game doesn't appear to be what is shown in the PR images.

**Comment by Yellows91** [comment]
I really like it but...warden MP where?

Warden for MPs feels like it needs to be added back as it gives MP its special touch of having one person MAINLY assigned to brig.

**Comment by Pyrodes** [comment]
Best change i've seen for MPs so far, silencer keep malding

**Comment by Yellows91** [comment]
A personal issue I have.

The green was bad and was removed which I approve of but...
why did you make it LIGHT RED (the same color of alpha squad) it makes it super hard to tell which is alpha and which is MP comms speaking.

Can we PLEASE get crimson red MP comms back

**Review by nauticall — APPROVED** [review]
okay, sprites look good now! approooooved

**Comment by hotimep** [comment]
if this removes the mp jacket atleast make co able to have the mp jacket bcuz it looks fresh as fuck!!!

**Comment by Sabitix** [comment]
cant they at least keep the old beret?

**Comment by Patrick81291** [comment]
Warden helped new players with brigging and guidance when there was no CMP.

**Pull request closed without merging**